benchmarks/bench.erl

```diff
@@ -10,20 +10,20 @@ run() ->
   C = dict:from_list([{name, "blue"}, {current, false}, {url, "#Blue"}]),
   Ctx1 = dict:store(item, [A, B, C], Ctx0),
   % Ctx1 = dict:new(),
-  CT = mustache:compile(complex, "../examples/complex.mustache"),
-  T0 = erlang:now(),
+  CT = mustache:compile(complex, "examples/complex.mustache"),
+  T0 = erlang:monotonic_time(),
   render(CT, Ctx1, ?COUNT),
-  T1 = erlang:now(),
-  Diff = timer:now_diff(T1, T0),
+  T1 = erlang:monotonic_time(),
+  Diff = erlang:convert_time_unit(T1 - T0, native, micro_seconds),
```

Comment:
^ No need for the legacy unit here (`micro_seconds`) - OTP 19, the new minimum required version as per this PR, deprecated those units and introduced the prettier ones (e.g. `microsecond`.)

Reply:
Cool. Thanks.